deepchem/utils/equivariance_utils.py

```diff
@@ -0,0 +1,174 @@
+import math
+import torch
+from typing import Optional
+
+
+def su2_generators(j: int) -> torch.Tensor:
+    """Get the generators of the SU(2) Lie algebra for a given quantum angular momentum.
+
```

Comment:
Can we add more scientific explanation here? Why do we want these generators? Think a few paragraphs of high quality documentation